rudder-agent/SPECS/rudder-agent.spec

```diff
@@ -149,7 +149,7 @@ Requires: pmtools
 %define use_system_lmdb false
 %endif
 
-%if 0%{?suse_version} && 0%{?suse_version} >= 1140
```

Comment:
This was was also OpenSuSE 11.4 specific (it is the other part of the first one)